packages/lib/src/components/internal/IbanInput/utils.ts

```diff
@@ -183,3 +183,9 @@ export const getNextCursorPosition = (cursor, iban, previousIban) => {
 
     return cursor;
 };
+
+/**
+ * @param electronicFormatIban -
+ * @returns countryCode string
+ */
+export const getCountryCode = electronicFormatIban => electronicFormatIban.slice(0, 2);
```

Comment:
`electronicFormatIban` has type `any`. It is better if we can define proper types whenever is possible. This allow us to later on force more strict typescript rules and have well typed codebase